docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1592,6 +1592,32 @@ Status Codes:
     **Stream details**:
     Similar to the stream behavior of `POST /container/(id)/attach` API
 
+### Exec Start (websocket)
```

Comment:
I removed it. Hoping this build will succeed on drone I have been having issues all day.